framework/src/media/audio/audio_manager.c

```diff
@@ -273,11 +273,43 @@ static audio_manager_result_t find_audio_card(audio_io_direction_t direct)
 		if (type == type_chr) {
 			pthread_mutex_init(&(card[card_id].card_mutex), NULL);
 			pthread_mutex_lock(&(card[card_id].card_mutex));
+
 			card[card_id].config[device_id].status = AUDIO_CARD_IDLE;
 			card[card_id].card_id = card_id;
 			card[card_id].device_id = device_id;
 			found_cards++;
 			medvdbg("Found an audio card, total card : %d id : %d device : %d\n", found_cards, card_id, device_id);
+
```

Comment:
this function is not for find card which supports processing.
It just make a list of exist card.

Reply:
Okay, reverted.